catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/docs/rabbitmq-component.adoc

```diff
@@ -292,37 +296,37 @@ camel exchange then they will be set on the RabbitMQ message.
 |===
 |Property |Value
 
-|`rabbitmq.ROUTING_KEY` |The routing key that will be used when sending the message
+|`rabbitmq_ROUTING_KEY` |The routing key that will be used when sending the message
 
-|`rabbitmq.EXCHANGE_NAME` |The exchange the message was received from
+|`rabbitmq_EXCHANGE_NAME` |The exchange the message was received from
 
-|`rabbitmq.EXCHANGE_OVERRIDE_NAME` |Used for force sending the message to this exchange instead of the endpoint configured name on the producer
+|`rabbitmq_EXCHANGE_OVERRIDE_NAME` |Used for force sending the message to this exchange instead of the endpoint configured name on the producer
 
-|`rabbitmq.CONTENT_TYPE` |The contentType to set on the RabbitMQ message
+|`rabbitmq_CONTENT_TYPE` |The contentType to set on the RabbitMQ message
 
-|`rabbitmq.PRIORITY` |The priority header to set on the RabbitMQ message
+|`rabbitmq_PRIORITY` |The priority header to set on the RabbitMQ message
 
-|`rabbitmq.CORRELATIONID` |The correlationId to set on the RabbitMQ message
+|`rabbitmq_CORRELATIONID` |The correlationId to set on the RabbitMQ message
 
-|`rabbitmq.MESSAGE_ID` |The message id to set on the RabbitMQ message
+|`rabbitmq_MESSAGE_ID` |The message id to set on the RabbitMQ message
 
-|`rabbitmq.DELIVERY_MODE` |If the message should be persistent or not
+|`rabbitmq_DELIVERY_MODE` |If the message should be persistent or not
 
-|`rabbitmq.USERID` |The userId to set on the RabbitMQ message
+|`rabbitmq_USERID` |The userId to set on the RabbitMQ message
 
-|`rabbitmq.CLUSTERID` |The clusterId to set on the RabbitMQ message
+|`rabbitmq_CLUSTERID` |The clusterId to set on the RabbitMQ message
 
-|`rabbitmq.REPLY_TO` |The replyTo to set on the RabbitMQ message
+|`rabbitmq_REPLY_TO` |The replyTo to set on the RabbitMQ message
 
-|`rabbitmq.CONTENT_ENCODING` |The contentEncoding to set on the RabbitMQ message
+|`rabbitmq_CONTENT_ENCODING` |The contentEncoding to set on the RabbitMQ message
```

Comment:
I am wondering, would it be better if you are changing these headers, to follow the Camel style naming, e.g: `CamelRabbitMqContentEncoding`? 
Also such a change, needs to be added in the migration guide as it is a breaking change for existing camel routes

Reply:
I didn't want to even had to touch this Constants class but had to change as having dots in the constant value isn't working (I think this is after the introduction of CAMEL-15394). 
Also such a change, needs to be added in the migration guide as it is a breaking change for existing camel routes
>> This is what @oscerd was also mentioning. Then, I wud prefer that existing constant value also works now. 
Can we please fix that issue? Then, I do not have to adapt from dot to underscore. Thanks

Reply:
> I didn't want to even had to touch this Constants class but had to change as having dots in the constant value isn't working (I think this is after the introduction of [CAMEL-15394](https://issues.apache.org/jira/browse/CAMEL-15394)).

I see, then it was already not following Camel headers format (with the dot), hence I'd recommend to change these to follow Camel's format e.g `CamelRabbitmqContentEncoding` and put a note into the migration guide for the users since we are at it. @oscerd @davsclaus WDYT?

Reply:
I'm ok with this. I think we can break the behavior until the next LTS